### PR TITLE
Remove kwarg `current_app=None`. 

### PR DESCRIPTION
**Pull request by brianjbuck**
It does not appear to do anything and breaks on Django 1.10

```
TypeError at /reports/6/summary/pdf/
__init__() got an unexpected keyword argument 'current_app'
Request Method:	GET
Request URL:	http://127.0.0.1:8000/reports/6/summary/pdf/
Django Version:	1.10
```

### CODE REVIEW

**Comment by halfnibble** [comment]
Please accept this pull request and update pypi. Thanks.

**Comment by halfnibble** [comment]
This fixes issue #116.

**Comment by johnraz** [comment]
Hi, sorry for the delay.
Do you mind updating the travis config so we are sure everything is find in Django 1.9 and 1.10 ?

**Comment by johnraz** [comment]
Looks like fixing tests for Django >=1.9 will require some more work. I'd say it should come prior to any further change regarding compatibility.

**Comment by brianjbuck** [comment]
Yes. Interesting though, I am currently running this in production on Python 3.5 and Django 1.10. It makes me think that the tests just need to be tweaked?

**Comment by halfnibble** [comment]
Yes,

I also find this interesting since I am using Buck’s branch on a Production Django 1.10 site with Python 2.7.

All the errors are “TemplateDoesNotExist” errors.

- Josh

**Comment by johnraz** [comment]
If you look at the travis logs (eg: [here](https://travis-ci.org/incuna/django-wkhtmltopdf/jobs/154565940)) you'll see that this is related to the content type app somehow.

It is probably loaded to soon or missing from the settings in the test run.

**Comment by brianjbuck** [comment]
Ok, tests are now passing. I added `contenttypes` explicitly in the `INSTALLED_APPS` setting as well as the new `TEMPLATES` dictionary for Django 1.10. I also had to modify the urls for static files and media in `templates/footer.html` but its probably better that way long-term.

File: wkhtmltopdf/tests/run.py [hunk 1]
```diff
@@ -29,6 +30,14 @@
     MEDIA_URL='/media/',
     STATIC_ROOT=os.path.join(DIRNAME, 'static'),
     STATIC_URL='/static/',
+    TEMPLATES = [  # For Django >= 1.10. Ignored is lower versions
```
Comment (johnraz) [inline wkhtmltopdf/tests/run.py]: Small typo:
> Ignored *if* lower versions

Comment (brianjbuck) [inline wkhtmltopdf/tests/run.py]: Good catch. Fixed.

**Comment by johnraz** [comment]
Fantastic job @brianjbuck, thanks a lot.
I'll merge as soon as the typo is fixed.
I don't have any access to the pypi repo though so I can't take action on that :/

**Comment by johnraz** [comment]
@maxpeterson: maybe you can help with the pypi release?

**Pull request merged**